server/utils/group-photo.js

```diff
@@ -176,11 +176,10 @@ const createGroupPhotoStream = async (urls) => {
   try {
     const groupPhoto = await createGroupPhoto(urls);
     console.log('Group Photo Processed');
-    const png = await groupPhoto.png({
-      compressionLevel: 5,
-      quality: 100,
+    const output = await groupPhoto.jpeg({
+      quality: 75,
     });
-    await png.toFile('./temp/group-photo.png');
+    await output.toFile('./temp/group-photo.png');
```

Comment:
just need ".jpeg" here, and i think we're good to go!

Reply:
Oohf, that would definitely help—thanks @clif-os. Fixed!